src/main/java/frc/robot/commands/auton/AutonOnePieceMiddle.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * This auton will be used for the sides of the field. It will place a cube on the top level and then back out of the community zone.
```

Comment:
This is good! Love how you made use of the comment section to explain your Auton code!